src/main/resources/db/changelog/v.0.1/db.changelog-vendor-locations.yaml

```diff
@@ -0,0 +1,29 @@
+databaseChangeLog:
+  - changeSet:
+      id: 8-create-vendor-locations-table
+      author: elizaveta_dubogryz
+      changes:
+        - createTable:
+            tableName: vendor_locations
+            columns:
+              - column:
+                  name: id
+                  type: UUID
+                  constraints:
+                    primaryKey: true
+              - column:
+                  name: vendor_id
+                  type: UUID
+                  constraints:
+                    nullable: false
+              - column:
```

Comment:
I believe here you need country id as well and just one of them will be populated (either country or city). For example you have vendor, who works with all cities in Belarus

Reply:
+